specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/cdnwebapplicationfirewall.json

```diff
@@ -0,0 +1,919 @@
+          "minimum": 0,
+          "exclusiveMinimum": false
+        },
+        "ruleGroupOverrides": {
+          "description": "Defines the rule overrides to apply to the rule set.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleGroupOverride"
+          }
+        }
+      }
+    },
+    "ManagedRuleGroupOverride": {
+      "description": "Defines a managed rule group override setting.",
+      "required": [
+        "ruleGroupName"
+      ],
+      "properties": {
+        "ruleGroupName": {
+          "description": "Describes the managed rule group within the rule set to override",
+          "type": "string"
+        },
+        "rules": {
+          "description": "List of rules that will be disabled. If none specified, all rules in the group will be disabled.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleOverride"
+          }
+        }
+      }
+    },
+    "ManagedRuleOverride": {
+      "description": "Defines a managed rule group override setting.",
+      "required": [
+        "ruleId"
+      ],
+      "properties": {
+        "ruleId": {
+          "description": "Identifier for the managed rule.",
+          "type": "string"
+        },
+        "enabledState": {
+          "description": "Describes if the managed rule is in enabled or disabled state. Defaults to Disabled if not specified.",
+          "type": "string",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "x-ms-enum": {
+            "name": "ManagedRuleEnabledState",
+            "modelAsString": true
+          }
+        },
+        "action": {
+          "description": "Describes the override action to be applied when rule matches.",
+          "$ref": "#/definitions/ActionType"
+        }
+      }
+    },
+    "ManagedRuleSetDefinitionList": {
+      "description": "List of managed rule set definitions available for use in a policy.",
+      "properties": {
+        "value": {
+          "description": "List of managed rule set definitions.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleSetDefinition"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to retrieve next set of managed rule set definitions."
+        }
+      }
+    },
+    "ManagedRuleSetDefinition": {
+      "description": "Describes a managed rule set definition.",
+      "properties": {
+        "properties": {
+          "description": "Describes managed rule set definition properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagedRuleSetDefinitionProperties"
+        },
+        "sku": {
+          "description": "The pricing tier (defines a CDN provider, feature list and rate) of the CdnWebApplicationFirewallPolicy.",
+          "$ref": "./network.json#/definitions/Sku"
```

Comment:
The file "./network.json" does not exist. This should probably be `"./cdn.json#/definitions/Sku"` as I can see that definition in cnd.json over [here](https://github.com/Azure/azure-rest-api-specs/pull/5885/files#diff-6d53702dc9b0abd52dedefbde8482955R4272).

Reply:
ping @sushantsing !!

Reply:
ping @sushantsingh  !!